src/mainPublicShareSidebar.js

```diff
@@ -42,7 +42,7 @@ Vue.use(Vuex)
 
 const pinia = createPinia()
 
-store.dispatch('setMainContainerSelector', '#talk-sidebar')
+store.dispatch('setMainContainerSelector', 'body')
```

Comment:
Do we need this state now, if it is always `body`?

Reply:
Set to undefined, so it's not affecting components now. Want to move a clean-up of that store/state to the follow-up PR, since it's touching too many files